Sources/DashX/DashXClient.swift

```diff
@@ -125,9 +131,21 @@ public class DashXClient {
           }
         }
     }
+    
+    func loadIdentity() {
+        let preferences = UserDefaults.standard
+        self.accountUid = preferences.string(forKey: Constants.USER_PREFERENCES_ACCOUNT_UID)
+        ConfigInterceptor.shared.identityToken = preferences.string(forKey: Constants.USER_PREFERENCES_IDENTITY_TOKEN)
+    }
 
     func reset() {
+        let preferences = UserDefaults.standard
+        
+        preferences.removeObject(forKey: Constants.USER_PREFERENCES_ACCOUNT_UID)
+        preferences.removeObject(forKey: Constants.USER_PREFERENCES_IDENTITY_TOKEN)
+        
         self.accountUid = nil
+        ConfigInterceptor.shared.identityToken = nil
```

Comment:
Do we need ConfigInterceptor? Why is this different from accountAnonymousUid which also needs to be sent in the network request?

Reply:
Yes, we need ConfigInterceptor to make network calls.  It is an instance of the type - of Apollo networking work(`ApolloInterceptor`).
In our current implementation`accountAnonymousUid` is passing as Input in Queries and Mutations,
but this `identityToken` will be set as appHeader for the Apollo client networking call as key `X-IDENTITY-TOKEN`.